astrocut/tests/test_cube_cut.py

```diff
@@ -390,6 +426,49 @@ def test_target_pixel_file(cube_file, ffi_type, tmp_path):
     tpf.close()
 
 
+def test_tica_cutout_error(tmp_path):
```

Comment:
If possible, in cases like this, i find it nice to write out the correct test now, and mark it with xfail. Then, when the behavior is working, we will notice (the test will start failing, and we can just remove the decorator and have it start to pass).

https://docs.pytest.org/en/7.1.x/how-to/skipping.html

Reply:
I've filled in the placeholder test now, Let me know what you think!

Reply:
okay, just to confirm, instead of using xfail here on a real tica cube, we're mocking the tica cube?

Reply:
Exactly. Creating a cube with TICA headers, but taking the ImageHDU from a "SPOC" cube that includes the error array.

Reply:
Okay, I was suggesting something different with xfail. The reason I was suggesting to use xfail is because it's obvious when the test starts working because the test will fail when the assertions all pass.

I'm fine with whatever approach is done here but there are tradeoffs to each. 

For instance, when mocking these TICA files with SPOC -- it could lead to obsolete tests -- we may never come back to update the tests, and with it not actually testing the real TICA data, we wouldn't know if something were broken.

Reply:
Ah okay, that makes sense. Do you have any thoughts about the best way to access the current TICA cubes? Ideally (I think) without using astroquery.mast.Tesscut?

Reply:
We don't necessarily need to access the TICA files here -- just write something that _will_ be able to access them when they are available on S3. If we know what their paths are going to be.

I'm also fine with what you have here, mocking the TICA file, as long as we do come back and point it at an actual TICA file later.

Reply:
Okay, I added a new test (test_s3_tica_cube_cut), marked with xfail, that tries to make a cutout from a TICA cube on the S3 bucket. That test will continue to xfail until we have *all* of the TICA cubes up on the cloud; it uses a Sector 27 cube, and that will be the last Sector that gets copied to the S3 bucket.

This test we're talking about here (test_tica_cutout_error) is intended to make sure that we can make cutouts from *both* the TICA cubes that do have error arrays (current) and the TICA cubes that do not have error arrays (coming soon), while we are in the process of updating the TICA cubes, since we expect this to take ~weeks.

Reply:
very nice! i'll take another look through this later today.